test/FBeetsRevenueSharer.test.ts

```diff
@@ -157,6 +274,7 @@ describe("BeetsBar", function () {
       locker.address,
       chef.address,
       farmPid,
+      INITIAL_FBEETSLOCKERSSHARE,
```

Comment:
might wanna give that some more underscores `FBEETS_LOCKER_SHARE` ?  for my old eyes :)